sdk/cost-management/Azure.ResourceManager.CostManagement/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure CostManagement management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Replace this with:
The Cost Management APIs provide the ability to explore cost and usage data via multidimensional analysis, where creating customized filters and expressions allow you to answer consumption-related questions for your Azure resources. These APIs are currently available for Azure Enterprise customers.